db/migrations/20201130231152-create-user.js

```diff
@@ -8,18 +8,19 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
-      name: {
+      userName: {
         allowNull: false,
-        type: Sequelize.STRING(150)
+        type: Sequelize.STRING(250),
+        unique:true
       },
       email: {
         allowNull: false,
-        type: Sequelize.STRING(75),
+        type: Sequelize.STRING(250),
         unique:true
       },
       password: {
         allowNull: false,
-        type: Sequelize.STRING(50),
+        type: Sequelize.BYTEA(250),
```

Comment:
Curious about this decision. Can you tell me more about why BYTEA and not STRING?

Reply:
Yeah, just realized we actually changed this again, so we will need to alter this pull request regardless. It is currently listed as STRING.BINARY. I'm fairly unclear on the actual datatype that needs to be stored in this location as the hashed password value. Would definitely appreciate some clarification here.

Reply:
Hashed passwords are just strings, so you can just use Sequelize.STRING here. Your `bcrypt` (or possibly whatever algorithm you are using) hashes will always be the same length, so you can make the character limit here based on the length of those hash strings.